File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest64929.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest64929 {

    @GetMapping("/BenchmarkTest64929")
    public void BenchmarkTest64929(@CookieValue("session_token") String sessionToken, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String cookieValue = sessionToken != null ? sessionToken : "";
        String data = String.join(" ", cookieValue.split("\\s+"));
        try {
            Integer.parseInt(data);
        } catch (Exception ignored) {
        }
        response.getWriter().print("{\"status\":\"ok\"}");
    }
}

File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_64929.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest64929():
    user_id = request.args.get('id', '')
    data = ' '.join(str(user_id).split())
    return jsonify({'status': 'ok'}), 200, {'X-Frame-Options': 'DENY', 'Content-Security-Policy': "frame-ancestors 'none'"}
